jenkins/opensearch-maven-release/publish-to-maven.jenkinsfile

```diff
@@ -16,7 +16,7 @@ pipeline {
     agent {
         docker {
             label 'Jenkins-Agent-AL2023-X64-M54xlarge-Docker-Host'
-            image 'opensearchstaging/ci-runner:ci-runner-centos7-opensearch-build-v3'
+            image 'opensearchstaging/ci-runner:centos7-x64-arm64-jdkmulti-node10.24.1-cypress6.9.1-20211130'
```

Comment:
Any reason we are changing the version? Maybe include the reason in PR description? or attach an issue?
Curious if anything is breaking and everyone using that library needs to bump

Reply:
We use 1.6.5 version of the maven plugin here: https://github.com/opensearch-project/opensearch-build/blob/main/publish/stage-maven-release.sh#L140

Which is very old back to 2016 that doesnt support jdk21: https://github.com/sonatype/nexus-maven-plugins/releases/tag/nexus-maven-plugins-1.6.5

Therefore we are switching back to the old image which has older version of jdk by default and is a known good version. Thanks.

Reply:
Yeah the time we updated this image doesn't go through real job as we weren't using main branch in Jenkins job.